crates/trigger/src/runtime_config/sqlite.rs

```diff
@@ -49,24 +49,35 @@ async fn execute_statements(
     if statements.is_empty() {
         return Ok(());
     }
-    let Some(default) = databases.get("default") else {
-        debug_assert!(
-            false,
-            "the 'default' sqlite database should always be available but for some reason was not"
-        );
-        return Ok(());
-    };
 
     for m in statements {
-        if let Some(file) = m.strip_prefix('@') {
+        if let Some(config) = m.strip_prefix('@') {
+            let config = config.trim();
+            let (file, database) = match config.split_once(':') {
+                Some((file, database)) if database.trim().is_empty() => (file.trim(), "default"),
```

Comment:
This would be the `@file.sql:` case?  That feels a bit like a syntax error to me but I don't feel strongly - not a blocker.